### ISSUE TITLE
[Feature Request] Add a way to copy/share an album

### ISSUE THREAD

**Issue by VinnyVynce** [xou816/spot#522]
While there is an option to link/share a song, Spot seems to lack a way to link an entire album, which is something I do fairly often.
A solution could be adding a button next to the "add to library" button in the album view, which would copy the album link into the clipboard.

**Comment by busterbeam** [comment]
Yep, another feature to add.  just because an application is "lightweight" doesn't mean it shouldn't have features hidden behind gestures and context-menus.  bloated UI is when there hundreds of buttons sitting on the "idle" interface.

**Comment by SarahRemus** [comment]
I will start working on this!
I will implement the option to copy the link of an album through a dot menu in the album-header. The "add to library button" could also be moved into the dot menu, if the goal is to achieve a clean UI, but I will not do this for now. 
I prefer adding a dot menu instead of a dedicated copy-button to make it possible to add more features for albums in the future.

**Comment by busterbeam** [comment]
thank you @SarahRemus !
